packages/measure/src/types.ts

```diff
@@ -25,4 +25,9 @@ export interface MeasureResults {
 
   /** Array of measured render/execution count for each run */
   counts: number[];
+
+  redundantRender: {
```

Comment:
```suggestion
  redundantRenders: {
```

Reply:
It would make sense to only add this for `measurePerformance` functio nand not for `measureFunction`. Therefore let's have 👍🏻 

```
interface MeasurePerfromanceResults extends MeasureResult {
  redundantRenders: RendundantRendersResults;
}

interface RendundantRendersResults {
  initialRenders: number;
  updates: number;
}

```